packages/marko-web-identity-x/utils/gtm-events.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef */
+export const loginSubmit = ({ event = 'login', email }) => {
+  console.log('gtm-event-login-submit: ', event);
+  const dataLayer = window.dataLayerIdentityX;
+  if (dataLayer) dataLayer.push({ event, email });
+};
+
+export const profileView = ({ event = 'profile_view' }) => {
+  console.log('gtm-event-profile-view: ', event);
+  const dataLayer = window.dataLayerIdentityX;
+  if (dataLayer) dataLayer.push({ event });
+};
+
+export const profileSubmit = ({ event = 'update_profile', email }) => {
```

Comment:
You made the default value in the Vue component `profile_submit`, but the default value here is `update_profile`. Which is correct?